www/about/policies/terms-of-service.md

```diff
@@ -0,0 +1,411 @@
+# Gratipay Terms of Service
+
+These Terms of Service ("Terms") are an agreement between you, a user
+("Participant") of the Gratipay service (the "Service"), and Gratipay, LLC
+("Gratipay"). They govern your use of the Service and your relationship with
+Gratipay. When you register as a Gratipay Participant, you agree to be bound by
+these Terms.
+
+If you have any questions about these terms, please contact us:
+
+Gratipay, LLC
+716 Park Road
+Ambridge, PA 15003
+USA
+legal@gratipay.com  
+
+1. Registration, Security, and Privacy
+
+  1. When you register for the Service, you agree to provide Gratipay with
+     complete and accurate registration information. You are responsible for
+keeping your account information, including your contact and payment
+information, up to date. Gratipay takes our Participants' privacy seriously. We
+will only use or share your information and Content as described in our Privacy
+Policy, which is incorporated into these Terms.
+
+  1. You are responsible for maintaining the security of your account
+     information, including your password. You agree to notify Gratipay
```

Comment:
We don't provide password authentication (see #1052; we do have API keys, which are basically passwords, but we only allow you to use them with Basic authentication, not cookie authentication. Basic auth is useful from programmatic clients that want to consume our JSON API, but not for human interaction with the site). All authentication via a web browser is done with OAuth against a social network (Twitter, Facebook, etc.).